sources/parsley/main.py

```diff
@@ -71,7 +75,7 @@ def main():
             last_heartbeat_time = now
             healthy = "Healthy" if time.time() - last_valid_message_time < 1 else "Dead"
             sender.send(HEARTBEAT_CHANNEL, {
-                "id": f"{gethostname()}/{args.format}", "healthy": healthy})
+                "id": f"{gethostname()}/{args.format}/{os.getpid()}", "healthy": healthy})
```

Comment:
I'd wrap this f"{gethostname()}/{args.format}/{os.getpid()}" in a function or variable, and replace all use later on with it.

Also, would port name be better than using pid? I am thinking if for some reason I need to restart parsley, then the dashboard will need to be updated again.